src/app/home/play-episode/play-episode.html

```diff
@@ -28,7 +28,7 @@
 <div class="ui container episode-info-container" *ngIf="!!episode">
     <h3 class=episode-title>
         <span class="bangumi-name">{{episode.bangumi.name_cn}}</span>
-        <span class="episode-no">{{episode.episode_no}}</span>
+        <span class="episode-no" type="padding-left:6px">{{episode.episode_no}}</span>
```

Comment:
`type` is not a valid attribute, you should use `style`
but, inline style is discouraged in production, unless you have a valid reason.
you should edit styles in the less file of the componentn